poetry.lock

```diff
@@ -120,11 +120,11 @@ python-versions = ">=3.6"
 
```

Comment:
Wondering why isn't this file ignored?

Reply:
[`poetry.lock` should be commited](https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control)

Although not required for libs (because it has no effects for builds), this is important so that anyone helping with development has the same dev environment, tests run with the same libs versions and etc.